src/cpp/fastdds/builtin/type_lookup_service/TypeLookupReplyListener.hpp

```diff
@@ -59,23 +61,46 @@ class TypeLookupReplyListener : public fastrtps::rtps::ReaderListener
     virtual ~TypeLookupReplyListener() override;
 
     /**
-     * @brief Method call when this class is notified of a new cache change
-     * @param reader The reader receiving the cache change
-     * @param change The cache change
+     * @brief Registers TypeIdentifier and TypeObject in TypeObjectRegistry.
+     * Also notifies all callbacks for the type and removes the current SampleIdentity from the list.
```

Comment:
```suggestion
     * This method also notifies all type related callbacks and removes the current SampleIdentity from the pending request list.
```

Reply:
fixed in [29681bc](https://github.com/eProsima/Fast-DDS/pull/4240/commits/29681bc3ecb5a0cf1f4e298b7bf093bff1ab1753)